solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -237,14 +237,12 @@ client.add(Arrays.asList(d1, d2));
 --
 
 == Query Time
-This is the Apache Solr query approach designed to support dense vector search:
 
-=== knn Query Parser
-The `knn` k-nearest neighbors query parser allows to find the k-nearest documents to the target vector according to indexed dense vectors in the given field.  The set of documents can be Pre-Filtered to reduce the number of vector distance calculations that must be computed, and ensure the best `topK` are returned.
+Apache Solr provides two query parsers that work with dense vector fields, that each support differnet ways of matching documents based on vector similarity: The `knn` query parser, and the `vecSim` query parser.
 
-The score for a retrieved document is the approximate distance to the target vector(defined by the similarityFunction configured at indexing time).
+Both parsers return scores for retrieved documents that is the approximate distance to the target vector (defined by the similarityFunction configured at indexing time) and both support "Pre-Filtering" the document graph to reduce the number of candidate vectors evaluated (with out needing to compute their vector similarity distances).
```

Comment:
typo? -> Both parsers return **a score** for retrieved documents that **is**...

Reply:
Hmmm... I think the existing wording is more grammatically correct? ... your change implies that both of the parsers return a _single_ score for _all_ of the documents?

Reply:
I was just worried about the discordance between the number of 'scores' and the verb 'is' ?


Maybe : "Both parsers return **scores** for retrieved documents that **are** the approximate distance to the target vector" ?

Reply:
Ah, ok ... yeah sorry, i was focused on your switch to singular that i didn't notice that dissonance with "is" later in the sentence.

I'll fix.